tools/apiview/emitters/typespec-apiview/package.json

```diff
@@ -53,19 +53,18 @@
     "dist/**",
     "!dist/test/**"
   ],
-  "dependencies": {
-    "@azure-tools/typespec-azure-core": "0.29.0",
-    "@azure-tools/typespec-autorest": "0.29.0",
-    "@typespec/compiler": "0.43.0",
-    "@typespec/http": "0.43.1",
-    "@typespec/rest": "0.43.0",
-    "@typespec/versioning": "0.43.0"
+  "peerDependencies": {
+    "@typespec/compiler": ">=0.40 <1.0",
+    "@typespec/versioning": ">=0.40 <1.0"
   },
   "devDependencies": {
-    "@typespec/eslint-plugin": "~0.43.0",
-    "@typespec/library-linter": "~0.43.0",
-    "@typespec/prettier-plugin-typespec": "^0.43.0",
-    "@typespec/eslint-config-typespec": "~0.6.0",
+    "@azure-tools/typespec-azure-core": ">=0.26 <1.0",
```

Comment:
The dev dependencies can be pinned(`~`) to the last you tested as this only matters to local developement

Reply:
I left the linter ones open but pinned the libraries.